src/mono/wasi/wasi.proj

```diff
@@ -122,14 +122,6 @@
   <Target Name="GenerateWasiPropsAndRspFiles">
 
     <!-- Generate wasm-props.json -->
-    <ItemGroup>
-      <!-- TODOWASI
-      <_WasiLinkFlags Include="-Wl, - - allow-undefined"/>
-      -Wl,- - export=malloc, - - export=free, - - export=__heap_base, - - export=__data_end \
```

Comment:
These exports don't make sense on WASI. They should be deleted from `WasiApp.targets` too that's for a separate change.